client/src/__tests__/Pages/Programmes/Programmes.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { AppConfiguration } from "../../../components";
+import { Programmes } from "../../../pages";
+import NewProgramme from "../../../pages/Programmes/NewProgramme";
+import ExistingProgramme from "../../../pages/Programmes/ExistingProgramme";
+import EmptyProgramme from "../../../pages/Programmes/EmptyProgramme";
+
+test("Main Programmes Page", () => {
```

Comment:
Please let me know if there's a better way to test these other than rendering each component separately.